x/gov/keeper/tally.go

```diff
@@ -73,7 +74,8 @@ func (keeper Keeper) Tally(ctx sdk.Context, proposal v1.Proposal) (passes bool,
 	})
 
 	// iterate over the validators again to tally their voting power
-	for _, val := range currValidators {
+	valL := maps.Values(currValidators)
+	for _, val := range valL {
 		if len(val.Vote) == 0 {
 			continue
 		}
```

Comment:
This code isn't sorting the values, so AFAICT doesn't produce behavior any different than ranging over the map directly?